canopen/profiles/p402.py

```diff
@@ -260,31 +260,31 @@ def _init_rpdo_pointers(self):
         for rpdo in self.rpdo.values():
             if rpdo.enabled:
                 for obj in rpdo:
-                    logger.debug('Configured RPDO: {0}'.format(obj.index))
+                    logger.debug('Configured RPDO: %s', obj.index)
```

Comment:
This should be formatted as hex `0x%04X` or using `pretty_index()` for consistency.

Reply:
Skipping this for the same reason the last commit was reverted